docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-slack.md

```diff
@@ -82,12 +82,12 @@ To create your Slack integration in Grafana Alerting, complete the following ste
 
 ## Next steps
 
-To add the contact point and integration you created to your default notification policy, complete the following steps.
+The Slack contact point is ready to receive alert notifications.
 
-1. Navigate to **Alerts & IRM** -> **Alerting** -> **Notification policies**.
-1. In the **Default policy**, click the ellipsis icon (…) and then **Edit**.
-1. Change the default policy to the contact point you created.
-1. Click **Update default policy**.
+To add this contact point to your alert:
 
-**Note:**
-If you have more than one contact point, add a new notification policy rather than edit the default one, so you can route specific alerts to Slack. For more information, refer to [Notification policies](ref:nested-policy).
+1. In Grafana, **navigate to Alerting** > **Alert rules**. Edit or create a new alert rule.
```

Comment:
```suggestion
1. In Grafana, navigate to **Alerting** > **Alert rules**. Edit or create a new alert rule.
```

Reply:
what's the edit or create a new alert? I think it should probably be a new step underneath